app/models/npc.rb

```diff
@@ -0,0 +1,51 @@
+class Npc < ActiveRecord::Base
+  class ConsequenceLimit < ActiveModel::Validator
+    def validate(npc)
+      if npc.consequences.length > 4
```

Comment:
Or perhaps you are building a ConsequenceLevels module for the National Broadcasting Corporation 😉